### PR TITLE
Adding West Virginia imagery map

### PR DESCRIPTION
**Pull request by fuji2086**
Updated to add the WV satellite imagery map

### CODE REVIEW

**Comment by Hipska** [comment]
This closes #73?

File: WME_OpenMaps.user.js [hunk 1]
```diff
@@ -12,10 +12,11 @@
 // @connect     clearly-dashboard.com
 // @connect     rj.gov.br
 // @connect     ccff02.minfin.fgov.be
+// @connect	wvgis.wvu.edu
```
Comment (Hipska) [inline WME_OpenMaps.user.js]: Could you use the same indentation?

**Comment by fuji2086** [comment]
Yes. It closed #73.

File: WME_OpenMaps.user.js (hunk 1 above)
Comment (fuji2086) [reply WME_OpenMaps.user.js]: Done.

**Pull request merged**